icicle/README.md

```diff
@@ -6,6 +6,7 @@
 mkdir -p build;
 cmake -DBUILD_TESTS=ON -DCURVE=<support_curve> -S . -B build;
 cmake --build build;
+./build/runner --gtest_brief=1;
 cd build && ctest; 
```

Comment:
oh - we need only one call ctest or runner, if runner fixed with include ordering then ctest needs to be removed